app/views/layouts/landing.html.erb

```diff
@@ -11,6 +11,7 @@
 
     <%= javascript_include_tag *webpack_asset_paths('babel-polyfill', extension: 'js') %>
     <%= javascript_include_tag *webpack_asset_paths('bootstrap', extension: 'js') %>
+    <%= javascript_include_tag *webpack_asset_paths('application', extension: 'js') %>
```

Comment:
Why is application.js being pulled in? With #90 being accepted, the styles should be present in the application. Bringing in application.js also pulls in rails-ujs and other things we don't want to pull into the SPA.